src/pages/user/profile.tsx

```diff
@@ -1,26 +1,51 @@
+import useAuthUser from '@/hooks/useAuthUser'
 import { Box, Container, Typography } from '@mui/material'
 import { NextPage } from 'next'
+import { useRouter } from 'next/router'
 
 interface Props {}
 
-const UserProfileDashboardPage: NextPage<Props> = ({}) => (
-  <Container>
-    <Box
-      sx={{
-        my: 4,
-        display: 'flex',
-        flexDirection: 'column',
-        justifyContent: 'center',
-        alignItems: 'center',
-      }}
-    >
-      <Typography variant="h6">User dashboard page</Typography>
-      <p>
-        This page shows the current user's application status. If a user has an
-        application that is in progress, it is visible here.
-      </p>
-    </Box>
-  </Container>
-)
+const UserProfileDashboardPage: NextPage<Props> = ({}) => {
+  // Redirect logic time
+  const { user, error } = useAuthUser()
+  const router = useRouter()
+
+  if (error) {
+    // TODO handle this more gracefully...
+    console.error(error)
+  }
+
+  if (!user) {
+    console.error('No user for this account exists!!')
+  }
+
+  // If the profile info is not set, then go to /user/info
+  if (!!user?.profile_info) {
```

Comment:
Isn't `profile_info` a boolean already? And isn't this checking if `profile_info===true` then redirect when it's supposed to be `profile_info===false && redirect()`

Reply:
also, what's the difference between `user/info` and `user/profile`?

Reply:
Good point on the logic lol.

Also, /profile is the dashboard and /info is the form I think. Maybe we should merge them?

Reply:
having `/profile` for the dashboard sounds a bit misleading imo. how about renaming `profile` to `dashboard`? and then using `profile` or `info` for the user profile form?